CHANGELOG.md

```diff
@@ -1,3 +1,21 @@
+# dbt_microsoft_ads v0.11.0
+
+## Schema Changes
+**5 total changes • 0 possible breaking changes
+| Table/Column                                      | Change Type | Old Name | New Name                                  | Notes                                                             |
+|---------------------------------------------------|-------------|----------|-------------------------------------------|-------------------------------------------------------------------|
+| microsoft_ads__campaign_country_report       | New Model   |          |  | New table that represents the daily performance of a campaign at the country/geographic region level.               |
+| microsoft_ads__campaign_region_report       | New Model   |          |  | New table that represents the daily performance of a campaign at the geographic region level.               |
```

Comment:
Oh also can you add links to the model's DAG/docs sites to the end model names

Reply:
Updated.